site/layouts/_default/baseof.html

```diff
@@ -4,6 +4,10 @@
 <head>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
+
+  {{ template "_internal/twitter_cards.html" . }}
+  {{ template "_internal/opengraph.html" . }}
```

Comment:
Do the "_internal/twitter_cards.html" and "_internal/opengraph.html" need to be added? They dont seem to be part of the repo or this PR...

Reply:
Ya, their Hugo internal components.